### PR TITLE
Add eventlog-machines/counters(-ekg)/recv

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: eventlog-counters/src/GHC/Eventlog/Counters.hs [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        RequestParGC -> return ()
+
+        -- stats
+        GCStatsGHC {..}                  ->  do
+            if gen == 0 then modifyTVar' (cntGcCount0 cnt) (+ 1) else modifyTVar' (cntGcCountS cnt) (+ 1)
+
+        HeapAllocated { allocBytes = x } -> writeTVar (cntHeapAlloc cnt) x
+        HeapSize { sizeBytes = x }       -> writeTVar (cntHeapSize cnt) x
+        BlocksSize { blocksSize = x }    -> writeTVar (cntBlocksSize cnt) x
+        MemReturn {}                     -> return () -- TODO
```
Comment (mpickering) [inline eventlog-counters/src/GHC/Eventlog/Counters.hs:140]: Still these TODO comments. Do you plan to add these events?

**Review by mpickering — APPROVED** [review]
Looks good, are you planning to add support for the two events you marked as TODO? `MemReturn`  would be a useful one to add.

File: eventlog-counters/src/GHC/Eventlog/Counters.hs (hunk 1 above)
Comment (phadej) [reply eventlog-counters/src/GHC/Eventlog/Counters.hs:140]: I'll add `MemReturn` but not `HeapInfoGHC` for now. it seems we should have the information latter provides already from other events.

Comment (phadej) [reply eventlog-counters/src/GHC/Eventlog/Counters.hs]: Done

**Pull request merged**